.github/workflows/tern-analysis.yml

```diff
@@ -0,0 +1,26 @@
+name: tern-analysis
```

Comment:
Could we have a more descriptive name? (I didn't know what "tern" is before looking it up.)

Reply:
What do you mean, surely it's obvious?!

But yeah, of course.